entrypoint.py

```diff
@@ -100,8 +93,8 @@ def modify_search_filter(cls, filter):
         return filter
 
     @classmethod
-    def apply(cls, qu):
-        """Default behavior is to not change a query at all."""
+    def modify_database_query(cls, _db, qu):
+        """Default behavior is to not change a database query at all."""
```

Comment:
`Facets` and `Pagination` now have separate methods for modifying a `Query` vs. a`Filter`, so I renamed `apply` to make it clear what the input is.